x-pack/plugins/task_manager/server/metrics/task_metrics_collector.ts

```diff
@@ -64,7 +64,7 @@ export class TaskManagerMetricsCollector implements ITaskEventEmitter<TaskLifecy
   private start() {
     if (!this.running) {
       this.running = true;
-      this.runCollectionCycle();
+      void this.runCollectionCycle();
```

Comment:
Apologies! I think it's not clear in the description:
`void` is used to intentionally leave the promise unhandled. It should be used with caution, but there are situations where it's fine to leave leave it unhandled because you know it will never reject. 
https://github.com/elastic/kibana/pull/181456#discussion_r1583763818

Reply:
I suspect devs will be confused when CI fails, but everything looks good locally - at least if they're like me and have VS Code set up and can run `scripts/type_check.js`.  We may want to include that invocation for `eslint_with_types` in the CI step, so if it does fail, folks will know how to test locally.

Reply:
++ I know @elastic/kibana-operations is tracking this in https://github.com/elastic/kibana/issues/117934